src/StatusQ/Controls/StatusIconTextButton.qml

```diff
@@ -0,0 +1,42 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Item {
```

Comment:
This feels again like it needs to be based on QQC2.AbstractButton (just customize the contentItem: https://doc.qt.io/qt-5/qtquickcontrols2-customize.html#customizing-button). Then you won't need any of the custom properties, signals or the MouseArea below

Reply:
I've started with your suggestion but faced to an issue when I want to change the default `cursorShape` to `Qt.PointingHandCursor`. The solution found is to add a custom `MouseArea` to change it so.. not sure which are the benefits in that case. Also if we could directly use the `text` or the `icon` from `Button`, it certainly would be an improvement, but we have our custom controls that define the style so I cannot see a benefit by doing this.
Do you know an alternative way of doing it simpler? Thanks!!

Reply:
AbstractButton is definitely the way to go as the base class, even regarding custom styling. You just have to override `background` and/or `contentItem`. As for the cursor shape, there is `HoverHandler`, too bad the `cursorShape` property has only been introduced in Qt 5.15 (https://doc.qt.io/qt-5/qml-qtquick-hoverhandler.html#cursorShape-prop) :/

Reply:
For now, you can still setup a temporary MouseArea to change the cursor shape (with a TODO to remove when we switch to Qt 5.15), sth like this:

```qml
MouseArea { 
  cursorShape: Qt.IBeamCursor
  acceptedButtons: Qt.NoButton
  hoverEnabled: true 
}
```